qualys-web-application-scanner/connector.py

```diff
@@ -1,5 +1,10 @@
-from connectors.core.connector import Connector, get_logger, ConnectorError
+""" Copyright start
```

Comment:
Change it to """
Copyright start
MIT License
Copyright (c) 2024 Fortinet Inc
Copyright end
"""

Reply:
updated